docs/source/admin_guide/breaking-upgrade.md

```diff
@@ -0,0 +1,160 @@
+# Upgrade - Redeployment for Breaking Changes
+
+For versions that are known to require redeployment when upgrading, the steps for upgrade include:
+
+- Back up user data by following [this guide](./backup.md).
+- Change existing cluster to a different URL (e.g. `qhub-old.mycompany.com`) so it is hidden.
+- Run `qhub upgrade` to make recommended modifications to the `qhub-config.yaml` file.
+- Deploy a new QHub with the desired version (to your original preferred URL e.g. `qhub.mycompany.com` but a new project_name to avoid resource name clashes).
+- Restore user data to the new cluster.
+- Destroy old cluster when happy.
+
+Please always check release notes for more details - and in all cases, backup your data before upgrading.
+
+**The rest of this guide assumes you are upgrading from version v0.3.14 to v0.4.**
```

Comment:
Should this be a note instead of a bold styling? we want to make this as visible as possible